specification/nhs-website-content-api.yaml

```diff
@@ -25,10 +25,10 @@ info:
 
     ## Related APIs
     The following APIs also give access to data from the NHS website:
-      - Service Search API
+      - [Service Search API](https://developer.api.nhs.uk/nhs-api#service-search-organisations-2)
 
     ## API status and roadmap
-    This API has moved from the nhs.uk developer API website. It is currently in development. If you are an existing user, you need to sign up for the API in the new [developer hub](https://digital.nhs.uk/developer).
+    This API has moved from the nhs.uk developer API website. It is currently [in development](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#statuses).
```

Comment:
I think this should say ...

This API has moved from the [NHS website developer portal](https://developer.api.nhs.uk/).